packages/client/tests/functional/extensions/model.ts

```diff
@@ -325,17 +324,10 @@ testMatrix.setupTestSuite(
         {
           badInput: true,
           ~~~~~~~~
-        + where: {
-        +   id?: String,
-        +   email?: String
-        + }
+        ? where?: UserWhereUniqueInput
```

Comment:
should this be recursively expanded as before?

Reply:
We don't get all the info from the engine for that.
Created an issue https://github.com/prisma/team-orm/issues/214